fern/fern.config.json

```diff
@@ -1,4 +1,4 @@
 {
   "organization": "fern",
-  "version": "0.29.0"
+  "version": "0.32.0"
```

Comment:
We'll want to revert this - there's actually a broken link in our docs that we can fix in a separate PR. Your change here won't require this version to be upgraded. This is the cause of [this CI failure](https://github.com/fern-api/fern/actions/runs/9994086972/job/27623041862).